docs/Style.thy

```diff
@@ -13,71 +13,85 @@ theory Style
 begin
 
 text \<open>
-This page describes the Isabelle syntax style conventions that we follow. For more on semantic
-style see Gerwin's style guides [1] and [2].
-\<close>
+  This page describes the Isabelle syntax style conventions that we follow. For more on semantic
+  style see Gerwin's style guides [1] and [2].\<close>
 
 section \<open>General Principles\<close>
 
 text \<open>
-Goals:
-  * Readability and consistency:
+  Goals:
+    * Readability and consistency:
       reduce cognitive overhead.
 
-  * Make good use of horizontal space:
+    * Make good use of horizontal space:
       most screens are wide, not high
 
-  * Semantic indentation:
+    * Semantic indentation:
       where possible group things according to meaning. But remain syntactically consistent:
       a sub-term should not go more left than its parent term.
 
-  * Support tools like diff and grep:
+    * Support tools like diff and grep:
       diff tries to find the "function" each chunk belongs to by indentation. If we remain
       compatible, diffs are much easier to read. Pull requests mean we read a lot of diffs.
       grep and other regexp search is not needed as much as it used to, but it's still often
       useful to be able to find constants and lemmas by regexp search outside of Isabelle
 
-  * Don't fight existing auto-indent where possible:
+    * Don't fight existing auto-indent where possible:
       there can be exceptions, but each time we go against defaults, it will be annoying to follow.
 
-  * Compatibility with upstream style where it exists:
+    * Compatibility with upstream style where it exists:
       there is not much consistent Isabelle distribution style, but where there is, we should follow
-      it, so that others can read our code and we can contribute upstream
-\<close>
+      it, so that others can read our code and we can contribute upstream\<close>
 
 text \<open>
-The main rules:
-  * Indent your proofs.
+  The main rules:
+    * Indent your proofs.
 
-  * Follow naming conventions.
+    * Follow naming conventions.
 
-  * Be aware that others have to read what you write.
-\<close>
+    * Be aware that others have to read what you write.\<close>
 
 text \<open>
-To not drive proof maintainers insane:
-  * Do not use `auto` except at the end of a proof, see [1].
+  To not drive proof maintainers insane:
 
-  * Never check in proofs containing `back`.
+    * Do not use `auto` except at the end of a proof, see [1].
```

Comment:
To save anyone else from my confusion, this is an artefact of the diff algorithm that github is using. There is no actual changed content in this list.